server/cherry_pick.go

```diff
@@ -92,7 +92,7 @@ func (s *Server) handleCherryPick(ctx context.Context, commenter, body string, p
 	command := getCommand(body)
 	args := strings.Split(command, " ")
 	mlog.Info("Args", mlog.String("Args", body))
-	if !pr.Merged.Valid || !pr.Merged.Bool {
+	if pr.Merged == nil || !*pr.Merged {
```

Comment:
Can we have helper methods for each of these fields? Like `IsMergedFalse`, `IsMilestoneNumberEmpty` which wraps the nil check and the false/0 check.

Reply:
I'll name them `GetFieldName()` similar to `github.PullRequest` e.g. `GetMerged()`